pallets/creditcoin/src/lib.rs

```diff
@@ -147,15 +146,10 @@ pub mod pallet {
 		fn close_deal_order() -> Weight;
 		fn exempt() -> Weight;
 		fn register_deal_order() -> Weight;
-		fn request_collect_coins() -> Weight;
-		fn persist_collect_coins() -> Weight;
-		fn fail_collect_coins() -> Weight;
 		fn remove_authority() -> Weight;
-		fn set_collect_coins_contract() -> Weight;
 		fn register_address_v2() -> Weight;
 		fn set_gate_contract() -> Weight;
 		fn set_gate_faucet() -> Weight;
```

Comment:
confirm with Dave if we want or not to remove those.

They are related to collect coin only, they are extrinsic to specify which account holds CTC for collect coin in cases of GATE being burned.